analyses/sun-study-slim/index.html

```diff
@@ -0,0 +1,296 @@
+<!DOCTYPE html>
```

Comment:
existing slimmed-down version of this extension has been moved to `sun-study-slim` without changes